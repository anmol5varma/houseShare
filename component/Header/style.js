export default {
    container: {
        flexDirection: 'row',
        backgroundColor: '#54b8ea',
    },
    title: {
        fontSize: 20,
        flex: 1,
        textAlign: 'center',
        color: '#fff',
        margin: 10,
    }
}