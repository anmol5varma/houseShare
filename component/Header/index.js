/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import React, { Component } from 'react';
import { Platform, StyleSheet, Text, View } from 'react-native';
import style from './style'

const styles = StyleSheet.create(style);

export default class Header extends Component {
    render() {
        return (
            <View style={styles.container}>
                <Text style={styles.title}>List of Properties</Text>
            </View>
        );
    }
}
