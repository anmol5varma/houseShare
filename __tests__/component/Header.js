/**
 * @format
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import 'react-native';
import React from 'react';
import Header from '../../component/Header';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
