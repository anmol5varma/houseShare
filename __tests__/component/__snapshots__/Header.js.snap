// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#54b8ea",
      "flexDirection": "row",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "flex": 1,
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    List of Properties
  </Text>
</View>
`;
